### PR TITLE
Feature/491 save skip

### PR DESCRIPTION
**Pull request by lloback**
Fixed the SAVE & CONTINUE button when the user is coming from the PREVIEW page.
The rules that was there was only diverting to REVIEW after the user skipped all the questions.
This, fixes that.

### CODE REVIEW

File: src/views/QualifyingTests/QualifyingTest/Question.vue [hunk 1]
```diff
@@ -114,7 +120,7 @@ export default {
       return false;
     },
     nextPage() {
-      if (this.isLastQuestion || this.hasStartedAllQuestions) {
+      if (this.isLastQuestion || this.hasStartedAllQuestions || this.isComingFromReview) {
```
Comment (warrensearle) [inline src/views/QualifyingTests/QualifyingTest/Question.vue:123]: hmm, why does `hasStartedAllQuestions` not work sufficiently here? 
If it did then we wouldn't need the `isComingFromReview` variable

**Review by warrensearle — CHANGES_REQUESTED** [review]

File: src/views/QualifyingTests/QualifyingTest/Question.vue (hunk 1 above)
Comment (lloback) [reply src/views/QualifyingTests/QualifyingTest/Question.vue:123]: `hasStartedAllQuestions` is working but the code from `isComingFromReview` is neat.
I would keep both.

Comment (lloback) [reply src/views/QualifyingTests/QualifyingTest/Question.vue:123]: Let me know your thoughts

**Review by warrensearle — APPROVED** [review]
👍

**Pull request merged**